### PR TITLE
[15.0][ADD] stock_picking_batch_operation_quick_change: New module

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

ping @carlosdauden @pilarvargas-tecnativa

### CODE REVIEW

File: stock_picking_batch_operation_quick_change/wizards/stock_picking_batch_wizard_view.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                    </group>
+                </group>
+                <footer>
+                    <button
+                        string="Apply"
+                        name="action_apply"
+                        type="object"
+                        class="oe_highlight"
+                    />
+                    or
```
Comment (pilarvargas-tecnativa) [inline stock_picking_batch_operation_quick_change/wizards/stock_picking_batch_wizard_view.xml]:
```suggestion
                 
```

LGTM, just a small change, removes the "or" from the view to follow the odoo design style.

**Comment by sergio-teruel** [comment]
Changes done

**Review by pilarvargas-tecnativa — APPROVED** [review]
thanks!

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**